sources/functions/mono

```diff
@@ -24,11 +24,11 @@ function mono_repo_update() {
     distribution=$(lsb_release -is)
     codename=$(lsb_release -cs)
     if [[ -f /etc/apt/sources.list.d/mono-xamarin.list ]]; then
-        if grep -q "5.18" /etc/apt/sources.list.d/mono-xamarin.list; then
+        if grep -q "6.8" /etc/apt/sources.list.d/mono-xamarin.list; then
             :
         else
-            echo "deb https://download.mono-project.com/repo/${distribution,,} ${codename}/snapshots/5.18/. main" > /etc/apt/sources.list.d/mono-xamarin.list
-            echo_log_only "Upgrading to mono 5.18 snapshot"
+            echo "deb https://download.mono-project.com/repo/${distribution,,} ${codename}/snapshots/6.8/. main" > /etc/apt/sources.list.d/mono-xamarin.list
```

Comment:
Should we update old installs with an updater?

Reply:
You mean this updater?

https://github.com/swizzin/swizzin/blob/master/scripts/update/mono.sh#L9

Reply:
slick